_data/pt-BR.yml

```diff
@@ -0,0 +1,294 @@
+  hCollusion: "Como se evita conluio entre mediadores, árbitros e negociadores?"
+  pCollusion: >
+    <p>Com o lançamento do novo protocolo de negociação na v1.2, os árbitros não mais tem uma das chaves do escrow multisig  Isso quer que os negociadores ficam com controle total sob seus fundos e não há espaço para conluio.</p><p> Mediadores e árbitros continuam sendo <a href="https://docs.bisq.network/user-dao-intro.html#ensure-honesty-in-high-trust-roles" target="_blank">posições com bonds</a>, porém, como não podem assinar uma transação de pagamento para resolver uma disputa, eles podem  <span class="note">aconselhar</span> como resolver a disputa e é importante que sejam responsáveis e justos enquanto fazem isso.</p>
+  hChargeback: "O que acontece se a pessoa comprando o bitcoin fizer um estorno depois de o bitcoin ter sido liberado pelo endereço multi-assinatura?"
+  pChargeback: >
+    <p>A Bisq só suporta meios de pagamento em que estornos não são fáceis ( por isso que PayPal e cartões de crédito não são suportados). Ainda há um risco pequeno de estorno com bancos. Se um banco executar um estorno após a liberação do BTC, não há nada que um árbitro ou mediador possam fazer.</p> <p> O objetivo da Bisq é fazer esse caso o mais inatrativo possível utilizando três mecanismos</a>:</p> <ol>
+      <li>Idade da conta faz com que contas de fiat recém criadas na Bisq tenham <a href='https://docs.bisq.network/payment-methods#supported-fiat-payment-methods' target="_blank">limites inferiores de negociação</a>, e os mesmos aumentam com o tempo.</li>
+      <li><a href='https://docs.bisq.network/payment-methods#account-signing' target="_blank">Assinatura de conta</a> faz com que meios de pagamento de alto risco sejam assinados ao verificar a integridade de um negociador antes que a idade da conta entre em vigor. Até que essas contas sejam assinadas, elas permanecem com o limite de 0.01 BTC.</li>
+      <li>Meios de pagamento fiat são removidos prontamente assim que descobrimos que são feitos estornos.</li>
+    </ol>
+  hReputationSystem: "Um sistema de reputação pura entre os negociadores não tornaria os mediadores e os árbitros desnecessários?"
+  pReputationSystem: >
+    <p>Reputação pura, sem a opção da arbitragem, é um sistema de proteção fraco, já que você nunca poderia evitar ataques do tipo <a href="https://en.wikipedia.org/wiki/Sybil_attack" target="_blank">sybil</a> ou <a href="https://en.wikipedia.org/wiki/Confidence_trick" target="_blank"> long con.</p> <p>Tais mecanismos também são problemáticos para proteger a privacidade e manter o serviço descentralizado.</p>
```

Comment:
```suggestion
    <p>Reputação pura, sem a opção da arbitragem, é um sistema de proteção fraco, já que você nunca poderia evitar ataques do tipo <a href="https://en.wikipedia.org/wiki/Sybil_attack" target="_blank">sybil</a> ou <a href="https://en.wikipedia.org/wiki/Confidence_trick" target="_blank"> long con<a/>.</p> <p>Tais mecanismos também são problemáticos para proteger a privacidade e manter o serviço descentralizado.</p>
```

Reply:
fixed with changes in transifex and applied here, see commit 
72292a5